flaskr/app.py

```diff
@@ -0,0 +1,36 @@
+import os
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
```

Comment:
SQLAlchemy for ORM and flask_migrate for DB migrations (no reply necessary, just making sure I understand it all)